examples/bootstrap_node.rs

```diff
@@ -0,0 +1,100 @@
+// Copyright 2019 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under the MIT license <LICENSE-MIT
+// http://opensource.org/licenses/MIT> or the Modified BSD license <LICENSE-BSD
+// https://opensource.org/licenses/BSD-3-Clause>, at your option. This file may not be copied,
+// modified, or distributed except according to those terms. Please review the Licences for the
+// specific language governing permissions and limitations relating to use of the SAFE Network
+// Software.
+
+//! Bootstrap node which acts as a relay for other client nodes. It collects the info of multiple
```

Comment:
Perhaps add a note that this bootstrap example is intended to be used with client_node.rs example.